Sources/NIO/SelectableEventLoop.swift

```diff
@@ -146,13 +146,22 @@ internal final class SelectableEventLoop: EventLoop {
         }
     }
 
+    public var isOpen: Bool {
+        switch self.externalState {
+            case .open, .closing:
```

Comment:
`execute` gets called when `self.externalState` = `.closing` [here](https://github.com/apple/swift-nio/issues/1383#issuecomment-585754801).  We *could* create an backdoor that one case, but I figured since there are currently no checks, this still covers what I imagine to be the majority of cases.

Reply:
I think `.closing` is fine: once we're actually _closed_ we want to reject new work, but while we're merely `closing` that's fine